Automattic/publisher-media-kit-for-newspack
# feat: update patterns and remove unused files

This PR takes the various patterns to the next level. Making the most of the Editor instead of relying on custom CSS.

![e.g](https://github.com/Automattic/publisher-media-kit-for-newspack/assets/177929/6bf35112-4350-461c-83a4-ef6cb330b805)

I've also removed a bunch of unused files.

Temp demo: https://speedily-given.wp.build/media-kit/

__How to test__

1- Install this plugin, activate it
2- Go to edit the new Media Kit page created (note: you will need to permanently delete any previously created)
3- Check if it works as expected (please use the block theme for now. if you use the classic theme, the colours will be missing)
